packages/fxa-profile-server/lib/routes/avatar/delete.js

```diff
@@ -31,16 +31,16 @@ module.exports = {
         .optional(),
     },
   },
-  handler: function deleteAvatar(req, reply) {
+  handler: async function deleteAvatar(req) {
     if (req.params.id === DEFAULT_AVATAR_ID) {
       // if we are clearing the default avatar then do nothing
-      return reply({});
+      return {};
     }
 
     const uid = req.auth.credentials.user;
     let avatar, lookup;
 
-    req.server.methods.profileCache.drop(uid, () => {
+    return req.server.methods.profileCache.drop(uid).then(() => {
```

Comment:
some of these above and below can be converted to `await` but to avoid more diffs we can do that later